custom_components/tuya_local/devices/fresco_hydrateultra_petfountain_alt.yaml

```diff
@@ -0,0 +1,233 @@
+        name: sensor
+        unit: s
+
+  - entity: sensor
+    name: Last Water change
+    icon: "mdi:water-circle"
+    category: diagnostic
+    class: duration
+    dps:
+      - id: 101
+        type: integer
+        name: sensor
+        range:
+          min: 0
+          max: 7200
+        unit: min
+
+  - entity: sensor
+    name: Water health
+    icon: "mdi:water-circle"
+    category: diagnostic
+    class: duration
+    dps:
+      - id: 101
+        type: integer
+        name: sensor
+        range:
+          min: 0
+          max: 7200
+        mapping:
+          - target_range:
+              min: 100
+              max: 0
+        precision: 1
+        unit: "%"
+
+  - entity: button
+    name: Reset Water health
+    icon: "mdi:sync"
+    category: config
+    dps:
+      - id: 101
+        type: integer
+        name: button
+        mapping:
+          - dps_val: 0
+            value: true
+
+  - entity: sensor
+    name: Last Filter change
+    icon: "mdi:air-filter"
+    category: diagnostic
+    class: duration
+    dps:
+      - id: 102
+        type: integer
+        name: sensor
+        range:
+          min: 0
+          max: 43200
+        unit: "m"
+
+  - entity: sensor
+    name: Filter health
+    icon: "mdi:air-filter"
+    category: diagnostic
+    class: duration
+    dps:
+      - id: 102
+        type: integer
+        name: sensor
+        range:
+          min: 0
+          max: 43200
+        mapping:
+          - target_range:
+              min: 100
+              max: 0
+        precision: 1
+        unit: "%"
+
+
+  - entity: button
+    name: Reset Filter health
+    icon: "mdi:sync"
+    category: config
+    dps:
+      - id: 102
+        type: integer
+        name: button
+        mapping:
+          - dps_val: 0
+            value: true
+
+  - entity: sensor
+    name: Last pump maintenance
+    icon: "mdi:pump"
+    class: duration
+    category: diagnostic
+    dps:
+      - id: 103
+        type: integer
+        name: sensor
+        range:
+          min: 0
+          max: 86400
+        unit: min
+
+  - entity: sensor
+    name: Pump health
+    icon: "mdi:pump"
+    class: duration
+    category: diagnostic
+    dps:
+      - id: 103
+        type: integer
+        name: sensor
+        range:
+          min: 0
+          max: 86400
+        mapping:
+          - target_range:
+              min: 100
+              max: 0
+        precision: 1
+        unit: "%"
+
+  - entity: button
+    name: Reset Pump health
+    icon: "mdi:sync"
+    category: config
+    dps:
+      - id: 103
+        type: integer
+        name: button
+        mapping:
+          - dps_val: 0
+            value: true
+
+  - entity: sensor
+    name: Working Time
+    icon: mdi:briefcase-clock
+    category: diagnostic
+    dps:
+      - id: 104
+        type: integer
+        name: sensor
+        range:
+          min: 0
+          max: 124002400
+
+  - entity: sensor
+    name: Bright Time
```

Comment:
It isn't clear what this is.  Is it something to do with the indicator?

Reply:
Basically LED runtime and same issue as above.